vignettes/customizing-dccvalidator.Rmd

```diff
@@ -63,6 +63,10 @@ default:
 ## Configuration options
 
 * `parent`: The Synapse project or folder where files will be stored
+* `path_to_markdown`: The `system.file()` path to the markdown document with app 
+  instructions. This option must follow the format 
+  `!expr system.file("path_to_md_document.Rmd", package = "dccvalidator")`. 
+  If you wish to omit app embedded instructions, insert `!expr NA`.
```

Comment:
I suggest something like "Location of an R Markdown document with app instructions. If you wish to omit instructions, insert `!expr NA`."